falcon/constants.py

```diff
@@ -25,7 +25,7 @@
     'PROPPATCH',
     'REPORT',
     'UNCHECKIN',
-    'UNLOCK'
+    'UNLOCK',
```

Comment:
Ouch, so it means what we were supporting before was actually `UNLOCKUPDATE`? :thinking:

Reply:
Yep. 💩